features/steps/room_steps.js

```diff
@@ -20,9 +21,11 @@ Given('a Workspace with an {publicityLevel} Room', function (publicityLevel) {
   return 'pending';
 });
 
-When("a {actor} unlocks the {room} with {roomKey}", function (actor, room) {
-  // Write code here that turns the phrase above into concrete actions
-  return "pending";
+When("a {actor} unlocks the {room} with {roomKey}", async function (actor, room, roomKey) {
+  room.name = "Listed Locked Room 1";
+  roomKey = "secret"
+  const roomSettingPage = await this.workspace.enterConfigureRoom(room, roomKey);
+  await roomSettingPage.unlock();
```

Comment:
Not blocking: Something that is indicating a rough edge/seam in our test framework is how we are overriding the room name like this.

I wonder if there's a way we could refactor our parameterTypes so that feature file can indicate whether the `Room` is locked or unlocked; perhaps this is done by composing the `Room` parameter type and the `AccessLevel` and `Visibility` parameter typ or perhaps the parameterType for Room becomes "greedier"?